### PR TITLE
feat: add initial support for import, provide minimal fmt

### PR DESCRIPTION
**Pull request by mvertes**
The `import` statement is now parsed. It only provides minimal support for the `fmt` package (only `Println` symbol is defined).

This should be sufficient to pass a few tests.

Full support of package namespaces, source and binary imports will be supported later, based on this work.

### CODE REVIEW

File: parser/symbol.go [hunk 1]
```diff
@@ -79,5 +81,7 @@ func initUniverse() map[string]*symbol {
 		"false": {index: unsetAddr, value: vm.ValueOf(false), Type: vm.TypeOf(false)},
 
 		"println": {index: unsetAddr, value: vm.ValueOf(func(v ...any) { fmt.Println(v...) })},
+
+		"fmt": {kind: symPkg, index: unsetAddr, pkgPath: "fmt"},
```
Comment (ajnavarro) [inline parser/symbol.go]: why is this needed here? does that mean that we have to add all packages here?

Comment (mvertes) [reply parser/symbol.go]: The `fmt` symbol is not really needed here. The `import` statement and the fact that a matching binary package matches in `package.go` should be sufficient to proceed.

This is an experiment to pre-declare some package symbols (mainly from standard library) so they do not need to be imported explicetly and can be used directly. REPL mode in yaegi allows that, but it is not mandatory and can be achieved in a better way.

I will change that, thanks for the remark.

**Review by ajnavarro — APPROVED** [review]
